tensorflow/lite/examples/python/ssd_object_detection.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""objection_detection for tflite"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+import time
+
+import numpy as np
+from PIL import Image
+import matplotlib.pyplot as plt
+import matplotlib.patches as patches
+
+from tensorflow.lite.python import interpreter as interpreter_wrapper
+
+def load_labels(filename):
+  my_labels = []
+  input_file = open(filename, 'r')
+  for l in input_file:
+    my_labels.append(l.strip())
+  return my_labels
+
+if __name__ == "__main__":
+  file_name = "/tmp/image2.jpg"
+  model_file = "/tmp/detect.tflite"
+  label_file = "/tmp/labelmap.txt"
+  input_mean = 127.5
+  input_std = 127.5
+  floating_model = False
+  show_image = False
+
+  parser = argparse.ArgumentParser()
+  parser.add_argument("--image", help="image to be classified")
```

Comment:
suggest to define default value under argument declaration and parse argument with short key as well.
same comment can be applicable for all argument.